crates/sui-indexer-alt/src/handlers/kv_transactions.rs

```diff
@@ -66,4 +67,19 @@ impl Handler for KvTransactions {
             .execute(conn)
             .await?)
     }
+
+    async fn prune(
+        &self,
+        from: u64,
+        to_exclusive: u64,
+        conn: &mut db::Connection<'_>,
+    ) -> Result<usize> {
+        // TODO: use tx_interval. `tx_sequence_number` needs to be added to this table, and an index
+        // created as its primary key is on `tx_digest`.
```

Comment:
No, let's keep this table as it is. Its schema needs to match the schema and indices need to match what is offered in the KV store as much as possible.